frontend/packages/kubevirt-plugin/integration-tests/tests/models/rhvImportWizard.ts

```diff
@@ -99,9 +91,13 @@ export class RhvImportWizard extends ImportWizard {
     await this.selectSourceVirtualMachine(sourceVMName);
```

Comment:
this await fails. Please make sure the elements are in focus and clickable

Reply:
This await should look like this inside:
```
  async selectSourceVirtualMachine(sourceVirtualMachine: string) {
    await selectOptionByText(rhvView.ovirtVmSelect, sourceVirtualMachine);
  }
```
Does it look the same in your code? If not - it will fail